src/open_inwoner/cms/cases/views/cases.py

```diff
@@ -144,38 +156,43 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         config = OpenZaakConfig.get_solo()
         case_service = CaseListService(self.request)
-        form = None
+        context["filter_form_enabled"] = config.zaken_filter_enabled
 
         # update ctx with open submissions, cases, form for filtering
```

Comment:
```suggestion
        # update ctx with open submissions and cases (possibly fitered)
```
we no longer add a form to the context